app/concerns/web_request_concern.rb

```diff
@@ -113,6 +113,7 @@ def faraday
       unless boolify(interpolated['disable_redirect_follow'])
         builder.use FaradayMiddleware::FollowRedirects
       end
+      builder.request :multipart
```

Comment:
It shouldn't the middleware only is called when one of the parameter [responds to `content_type`](https://github.com/lostisland/faraday/blob/master/lib/faraday/request/multipart.rb#L29).